plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -91,6 +91,8 @@ func ClusterRoles() []rbac.ClusterRole {
 			Rules: []rbac.PolicyRule{
 				// TODO add future selfsubjectrulesreview, project request APIs, project listing APIs
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("selfsubjectaccessreviews").RuleOrDie(),
+
+				rbac.NewRule(Read...).Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
```

Comment:
I went ahead and introduced a new _ClusterRole_: "system:cluster-facilities." Please take a look at af4fb8aa220bdb3222a4b75ff217d58a8d48c09d and let me know
1. Whether you like the use of a separate _ClusterRole_, and
2. Whether you like the name, or have a better suggestion.